src/module/dice.js

```diff
@@ -149,6 +150,17 @@ export class DiceSFRPG {
                 let mult = 2;
                 roll.alter(add, mult);
                 flavor = `${title} (Critical)`;
+
+                if (criticalData != undefined) {
```

Comment:
The projects preference is to use the `!==` operator whenever possible rather than the dubious `!=`. Same goes for `===` vs `==`. Please correct.